source/core/TypeAndDataManager.cpp

```diff
@@ -1776,6 +1785,8 @@ NIError ArrayType::CopyData(const void* pData, void* pDataCopy)
             pDestElt += stride;
         }
     }
+    if (_isDataItem)
```

Comment:
Should we also do this for "ClusterType::CopyData"?

Reply:
See previous comment.